src/SFML/Graphics/RenderTextureImplFBO.cpp

```diff
@@ -32,23 +32,24 @@
 #include <SFML/System/Lock.hpp>
 #include <SFML/System/Err.hpp>
 #include <utility>
-#include <set>
+#include <unordered_map>
+#include <unordered_set>
 
 
 namespace
 {
     // Set to track all active FBO mappings
     // This is used to free active FBOs while their owning
     // RenderTextureImplFBO is still alive
-    std::set<std::map<sf::Uint64, unsigned int>*> frameBuffers;
+    std::unordered_set<std::unordered_map<sf::Uint64, unsigned int>*> frameBuffers;
 
     // Set to track all stale FBOs
     // This is used to free stale FBOs after their owning
     // RenderTextureImplFBO has already been destroyed
     // An FBO cannot be destroyed until it's containing context
     // becomes active, so the destruction of the RenderTextureImplFBO
     // has to be decoupled from the destruction of the FBOs themselves
-    std::set<std::pair<sf::Uint64, unsigned int> > staleFrameBuffers;
+    std::unordered_map<sf::Uint64, unsigned int> staleFrameBuffers;
```

Comment:
I will likely revert this one, it is a bit iffy. When I checked, the `Uint64` is the ID of the context. I assumed that each context can have one framebuffer but I now I see that might not be the right assumption. 

What I could do instead is also change this to:

```cpp
std::unordered_map<sf::Uint64, std::unordered_set<unsigned int>>
```

What do you think?

Reply:
Would this nested data structure give us an actual benefit? If we only need it for lookup of (contextId, frameBufferId) pairs, the existing one is easier.

Also, I'd be careful about having too many not-directly-related changes in this PR; it makes it harder to hunt down bugs (`git bisect`) in case we break something.